Sources/Swiftly/ListAvailable.swift

```diff
@@ -28,9 +28,9 @@ struct ListAvailable: SwiftlyCommand {
         The installed snapshots for a given devlopment branch can be listed by specifying the branch as the selector:
 
             $ swiftly list-available main-snapshot
-            $ swiftly list-available 5.7-snapshot
+            $ swiftly list-available 6.0-snapshot
 
-        Note that listing available snapshots is currently unsupported. It will be introduced in a future release.
+        Note that listing available snapshots before 6.0 is unsupported.
```

Comment:
I assumed this is because the data isn't available at the moment. Could this change? I don't think this text is necessary. I think the response when there are no snapshots available should make it clear.